src/sidebar/components/test/thread-list-test.js

```diff
@@ -103,6 +103,25 @@ describe('ThreadList', () => {
     wrapper.setProps({});
   };
 
+  context('invalid scroll container', () => {
```

Comment:
There is a simpler way to make the scroll container invalid that doesn't require installing a `querySelector` mock:

```js
  context('invalid scroll container', () => {
    it('should throw an error if the scroll container is missing', () => {
      fakeScrollContainer.remove();
      assert.throws(() => {
        createComponent();
      }, 'Scroll container is missing');
    });
  });
```